test/unit/fineract_client.test.ts

```diff
@@ -0,0 +1,112 @@
+/*****
```

Comment:
![image](https://github.com/mojaloop/mifos-core-connector/assets/16463615/afe66e2a-c084-404d-a371-b4be9f10b4f6)

Why do we have 3 different approaches for file naming: `camelCase`, `snake_case` and `kebab-style`?